bin/gwdetchar-scattering

```diff
@@ -263,7 +263,7 @@ for i, seg in enumerate(statea):
                  obs=args.ifo[0], verbose=msg, nproc=args.nproc).resample(128))
 
 scatter_segments = DataQualityDict()
-
+actives = SegmentList()
```

Comment:
@siddharth101, this is _very_ pedantic and I apologize in advance, but can you put an empty line after this?

```suggestion
actives = SegmentList()

```

My reasoning is for readability: it visually groups the definition with that of `scatter_segments` above, separate from the `for` loop below.

Reply:
Done. Thanks.

Reply:
No problem, thank you for humoring me!